tests/foreman/cli/test_hammer.py

```diff
@@ -183,16 +189,81 @@ def test_positive_disable_hammer_defaults(self):
             Defaults.add({'param-name': 'organization_id', 'param-value': default_org['id']})
             # Verify --organization-id is not required to pass if defaults are set
             result = ssh.command('hammer product list')
-            self.assertEqual(result.return_code, 0)
+            assert result.return_code == 0
             # Verify product list fail without using defaults
             result = ssh.command('hammer --no-use-defaults product list')
-            self.assertNotEqual(result.return_code, 0)
-            self.assertFalse(default_product_name in "".join(result.stdout))
+            assert result.return_code != 0
+            assert not (default_product_name in "".join(result.stdout))
             # Verify --organization-id is not required to pass if defaults are set
             result = ssh.command('hammer --use-defaults product list')
-            self.assertEqual(result.return_code, 0)
-            self.assertTrue(default_product_name in "".join(result.stdout))
+            assert result.return_code == 0
+            assert default_product_name in "".join(result.stdout)
         finally:
             Defaults.delete({'param-name': 'organization_id'})
             result = ssh.command('hammer defaults list')
-            self.assertTrue(default_org['id'] not in "".join(result.stdout))
+            assert default_org['id'] not in "".join(result.stdout)
+
+    @tier1
+    def test_positive_check_debug_log_levels(self):
+        """Enabling debug log level in candlepin via hammer logging
+
+        :id: 029c80f1-2bc5-494e-a04a-7d6beb0f769a
+
+        :expectedresults: Verify enabled debug log level
+
+        :CaseImportance: Medium
+
+        :BZ: 1760773
+        """
+        ssh.command("hammer admin logging --all --level-debug")
+        # Verify value of `log4j.logger.org.candlepin` as `DEBUG`
+        result = ssh.command("grep DEBUG /etc/candlepin/candlepin.conf")
+        assert result.return_code == 0
+
+        ssh.command("hammer admin logging --all --level-production")
+        # Verify value of `log4j.logger.org.candlepin` as `WARN`
+        result = ssh.command("grep WARN /etc/candlepin/candlepin.conf")
+        assert result.return_code == 0
+
+    @tier1
+    def test_positive_check_errata_dates(self):
```

Comment:
I understand bugzilla was assigned to component Hammer, and actually required fix in hammer-related code, but honestly I would expect that in specific component file - `tests/foreman/cli/test_errata.py`.

Similar to bz#1421298, which also is assigned to Hammer, but tests resides in `tests/foreman/cli/test_repository.py`